### PR TITLE
Add TC's for Request Form

### PR DESCRIPTION
**Pull request by VictoriaBordian**

### CODE REVIEW

File: packages/tests/features/requestForm.feature [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    Then  Request Form is split according to the arguments number required
+
+Scenario: Submit with a valid request (check the model of data received)
+    Given Request Form web component
+    And   a valid input for specific language
+    And   Submit button is enabled
+    When  user click on Submit button
+    Then  request is sent
+    And   object received contain the following data
+          |data       |
```
Comment (dmitriyslaym) [inline packages/tests/features/requestForm.feature]: do we need this data here? It feels like it's an additional property of our response

Comment (dmitriyslaym) [reply packages/tests/features/requestForm.feature]: do you think, that we need to include all the possible data types to check that they are parsed correctly? Should it be specified in the test scenario?

**Review by dmitriyslaym — COMMENTED** [review]
I would also like to add the scenario when a user has already typed something in the first input and clears the args count input to type a different number - this first editor with it's content should not be removed.

File: packages/tests/features/requestForm.feature (hunk 1 above)
Comment (IrinaComindaru) [reply packages/tests/features/requestForm.feature]: I think why not
What are the possible data types?

Comment (dmitriyslaym) [reply packages/tests/features/requestForm.feature]: Null, undefined, number, string, object, array, function

Comment (IrinaComindaru) [reply packages/tests/features/requestForm.feature]: @dmitriyslaym do we expect the request to be sent successfully in any case? or for some of these data types we should receive an error?

Comment (dmitriyslaym) [reply packages/tests/features/requestForm.feature]: @IrinaComindaru All of the data types should be supported.

**Review by dmitriyslaym — APPROVED** [review]

**Pull request merged**